### PR TITLE
Add progress number

### PR DESCRIPTION
**Pull request by wvangeit**
Show the user the number of tasks (out of how many) have finished

### CODE REVIEW

File: .osparc/osparc-meta-parallelrunner/metadata.yml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 name: ParallelRunnerService
 description: "ParallelRunnerService"
 key: simcore/services/dynamic/osparc-meta-parallelrunner
-version: 0.0.10
-integration-version: 0.0.10
+version: 0.0.11
+integration-version: 2.0.0
```
Comment (wvangeit) [inline .osparc/osparc-meta-parallelrunner/metadata.yml:5]: @elisabettai better? ;-)

Comment (elisabettai) [reply .osparc/osparc-meta-parallelrunner/metadata.yml:5]: Indeed! 😉

File: Makefile [hunk 2]
```diff
@@ -49,6 +49,7 @@ validation_client_run: validation-clean
 	VALIDATION_CLIENT_INPUT_PATH=validation-tmp/outputs/output_1 VALIDATION_CLIENT_OUTPUT_PATH=validation-tmp/inputs/input_2 python validation-client/client.py
 
 docker_compose: validation-clean
+	docker-compose down
```
Comment (elisabettai) [inline Makefile]: Minor: `docker-compose` command I think has been deprecated in favor of `docker compose`. I am not sure if this is a problem for this specific service (I guess the CI successfully builds it anyway).

**Review by elisabettai — APPROVED** [review]

File: Makefile (hunk 2 above)
Comment (wvangeit) [reply Makefile]: Completely true, I was mirroring what was already there in the original Makefile. But I've replaced now all docker-compose with docker compose.

**Pull request merged**